app/cafe.py

```diff
@@ -0,0 +1,26 @@
+from datetime import date
+from app.errors import (
+    NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:

```

Comment:
From a clean code perspective, it's good to have a docstring for your classes to explain what they do. In this case, you can add a docstring to the `Cafe` class to explain its purpose. This will make your code easier to understand for other developers.